app/models/comment.rb

```diff
@@ -39,6 +39,11 @@ class Comment < ActiveRecord::Base
       limit(12).
       select([:id, :node_id, :title])
   }
+  scope :latest, -> {
+    where(["created_at >= ?", Date.current - 7.day])
```

Comment:
```suggestion
    where(["created_at >= ?", 7.days.ago])
```

Reply:
Thanks I didn't know this rails methods.